apollo-router/src/plugins/coprocessor/mod.rs

```diff
@@ -282,6 +301,7 @@ pub(super) struct SubgraphResponseConf {
 struct Conf {
     /// The url you'd like to offload processing to
     url: String,
+    experimental_http2: Option<CoprocessorHttp2Config>,
```

Comment:
could you put the option under a `traffic_shaping` section? Since we will probably extend it with more options, this will allow us to put them in the same place (and then later easily extend to stage specific config).
Also, could this use the [`Http2Config` from the traffic shaping plugin](https://github.com/apollographql/router/blob/b4dda0cf5739fae58313e25dd04ea64159b6a717/apollo-router/src/plugins/traffic_shaping/mod.rs#L77-L85)? This way the config schema would refer to the existing type instead of adding a new one

Reply:
ROFL, I had done that first, but then thought you might want to de-couple from that.  I'll return back to not having the separate config.

Reply:
traffic_shaping section is now added, now using Http2Config from traffic shaping plugin

Reply:
@Geal  -- Let me know next steps/what else we can do to get this over the finish line.